src/protocol/java/src/main/java/org/apache/jmeter/protocol/java/sampler/JavaSampler.java

```diff
@@ -121,9 +121,9 @@ private void initClass() {
             javaClass = Class.forName(name, false, Thread.currentThread().getContextClassLoader());
             Method method = javaClass.getMethod("teardownTest", JavaSamplerContext.class);
             isToBeRegistered = !method.getDeclaringClass().equals(AbstractJavaSamplerClient.class);
-            log.info("Created class: {}. Uses tearDownTest: ", name, isToBeRegistered);
+            log.info("Created class: %s. Uses tearDownTest: %b", name, isToBeRegistered);
```

Comment:
slf4j uses `{}` placeholders rather than `%..` ones.

```suggestion
            log.info("Created class: {}. Uses tearDownTest: {}", name, isToBeRegistered);
```